app/clients/db/mongodb_client.py

```
@@ -1 +1,35 @@
+        try:
+            return self.db[collection_name]
+        except Exception as e:
+            logger.error(f"Failed to get collection {collection_name}: {str(e)}")
+            raise RuntimeError(f"Failed to get collection {collection_name}: {str(e)}")
+
+    def close(self):
+        try:
+            self.client.close()
+        except Exception as e:
+            logger.error(f"Failed to close MongoDB connection: {str(e)}")
+            raise RuntimeError(f"Failed to close MongoDB connection: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)